string_to_string.go

```diff
@@ -0,0 +1,140 @@
+package pflag
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+// -- stringToString Value
+type stringToStringValue struct {
+	value   *map[string]string
+	changed bool
+}
+
+func newStringToStringValue(val map[string]string, p *map[string]string) *stringToStringValue {
+	ssv := new(stringToStringValue)
+	ssv.value = p
+	*ssv.value = val
+	return ssv
+}
+
+// Format: a=1,b=2
+func (s *stringToStringValue) Set(val string) error {
+	ss := strings.Split(val, ",")
```

Comment:
Should this actually be a CSV decode, instead of just splitting on ,?  What is there is a comma in the value?